Source/Core/DolphinQt/Config/Graphics/AdvancedWidget.cpp

```diff
@@ -338,11 +341,18 @@ void AdvancedWidget::AddDescriptions()
   static const char TR_LOAD_GRAPHICS_MODS_DESCRIPTION[] =
       QT_TR_NOOP("Loads graphics mods from User/Load/GraphicsMods/.<br><br><dolphin_emphasis>If "
                  "unsure, leave this unchecked.</dolphin_emphasis>");
-  static const char TR_INTERNAL_RESOLUTION_FRAME_DUMPING_DESCRIPTION[] = QT_TR_NOOP(
-      "Creates frame dumps and screenshots at the internal resolution of the renderer, rather than "
-      "the size of the window it is displayed within.<br><br>If the aspect ratio is widescreen, "
-      "the output image will be scaled horizontally to preserve the vertical resolution.<br><br>"
-      "<dolphin_emphasis>If unsure, leave this unchecked.</dolphin_emphasis>");
+  static const char TR_FRAME_DUMPS_RESOLUTION_TYPE_DESCRIPTION[] = QT_TR_NOOP(
+      "Selects how frame dumps (videos) and screenshots are going to be captured.<br>"
+      "If the video resolution changes during a recording, multiple video files will be created."
+      "<br><br><b>Window Resolution</b>: Uses the output window resolution (without black bars)."
+      "<br><b>Aspect Ratio Corrected Internal Resolution</b>: Uses the Internal Resolution (XFB "
+      "size), and corrects it by the target aspect ratio."
+      "<br><b>Raw Internal Resolution</b>: Uses the Internal Resolution (XFB size)"
+      " without correcting it with the target aspect ratio, "
+      "thus dumps will appears more crips.<br>This is "
```

Comment:
There's a typo here, but I have a suggestion for an alternative.

> Raw Internal Resolution: Uses the Internal Resolution (XFB size) without correcting it with the target aspect ratio. This will provide a dump without any correction so users can apply their own in editing software."

I wanted to say "video editing software" but this applies to screenshots too, so, yea.

Reply:
I'm not really worried about this, captured videos and screenshots are more "crisp", thus applying any kind of post processing or scaling on them in any external software will be easier. I think it's implied, and not something dolphin needs to specify.

Reply:
Well, the point of the option is that we're not applying any correct so video editors can apply their own. You are talking about visual quality a lot, and only implying the actual use case.

Furthermore you an signaling pretty strongly that raw is the better and correct option in all scenarios by applying positive language ("crisp" and "highest quality possible") to this option only. If a user doesn't know which option to go with and reads all three, they will pick the only one with positive language attached every time, regardless of whether it is the right fit for them and their use case.

IMO, this is a misleading description. Instead, you should just say what it is and what it is for, and that's it, like the other options.

Reply:
![image](https://github.com/dolphin-emu/dolphin/assets/7011366/98ea7635-d010-41c9-921b-f9de9701b034)

Reply:
> Raw Internal Resolution: Uses the Internal Resolution (XFB size) without correcting it with the target aspect ratio, allowing any kind of scaling or post processing from external editing software to be applied with a higher quality.

Ok, better. But I have some more notes.

Having Dolphin do aspect ratio correction doesn't _prevent_ scaling or post processing in external editing software, as is implied by this description. Raw doesn't "allow" it. Also, grammatically, it should be "at higher quality" not "with a higher quality".

I still take issue with quality being mentioned here. This option basically has Dolphin go hands off so video editors have completely control over the pixels of the footage. This could allow them to optimize the pixel clarity of their video, or they just won't apply pixel aspect ratio correction at all and _get a worse result_ than if they had used one of our correction options (which IMO is pretty likely). The whole point of this is that we're giving the user a clean slate for them to manipulate however they so please, which can mean basically anything. So I'm not super happy with saying that this option will give improve quality. Cause it may very well not.